workflow.txt

```diff
@@ -1,10 +1,9 @@
+fork on github
 git clone
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
+git checkout master
 git add .
+git commit -m ""
 git pull upstream master
```

Comment:
Oh ya, I did that for the merge exercise yesterday. don't know how I missed that. Thanks.